plugins/obs-webrtc/webrtc/Cargo.lock

```diff
@@ -0,0 +1,2716 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Don't add `lock` files to the repository - they are automatically generated and host-specific.

Reply:
I'm not sure I follow, but they shouldn't be host specific. They'll include all dependencies regardless of whether they match the host (target host specific dependencies) or not. Not including the lock would let transitive dependencies on a per-build shift, which seems unideal.

Reply:
After reading through https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html this seems indeed to be necessary for Rust, as transitive dependencies could specify `*` as their dependency versions, so full version pinning is only possible by checking in the `lock` file.

Annoying, but that's the Rust way.